src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPage.tsx

```diff
@@ -28,8 +29,11 @@ const useStyles = makeStyles(
     },
     content: {
       overflowX: "visible",
-      overflowY: "hidden",
-      width: 800
+      [theme.breakpoints.up("md")]: {
+        position: "absolute",
+        width: `calc(100vw - ${drawerWidthExpanded + theme.spacing(6)}px)`,
```

Comment:
```suggestion
        width: `calc(100vw - ${drawerWidthExpanded}px - ${theme.spacing(6)}px)`,
```
Makes it easier to switch spacing unit to rems in future